Generals/Code/Libraries/Source/WWVegas/WW3D2/dx8wrapper.cpp

```diff
@@ -274,10 +311,22 @@ void DX8Wrapper::Shutdown(void)
 		Release_Device();
 	}
 
-	for (int i = 0; i < MAX_TEXTURE_STAGES; i++) {
-		if (Textures[i]) {
-			Textures[i]->Release();
-			Textures[i] = NULL;
+	if (D3DInterface) {
+		D3DInterface->Release();
```

Comment:
Just Note: Zero Hour developer made mistake. This function now has 2 calls to `D3DInterface->Release` with the other being obsolete.